useful-scripts/misc-test-utils/src/services/methodologyService.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+import faker from 'faker';
+import adminApi from '../utils/adminApi';
+
+const { ADMIN_URL } = process.env;
+
+const methodologyService = {
+  createMethodology: async (publicationId: string): Promise<string> => {
+    const res = await adminApi.post(
+      `/api/publication/${publicationId}/methodology`,
+    );
+    console.log(
```

Comment:
Not using chalk here?  If you did you could get rid of the eslint-disable, but there might be a reason why you're not?